yo/db.py

```diff
@@ -263,123 +283,81 @@ def _create_notification(self, table=None, **notification):
                                  notification)
             return False
 
-    def create_wwwpoll_notification(self, **notification):
-        """ Creates a new notification in wwwpoll table
-
-        Keyword Args:
-            nid(int):
-            notify_type(str):
-            to_username(str):
-            from_username(str):
-            json_data(str):
-            created(datetime):
-            updated(datetime):
-            read(bool):
-            shown(bool):
-
-        Returns:
-            bool:             True on success, False otherwise
-        """
-        table = wwwpoll_table
-        return self._create_notification(table=table, **notification)
-
-    def create_notification(self, **notification):
-        """ Creates a new notification in the notifications table
-
-        Keyword Args:
-            nid(int):
-            notify_type(str):
-            to_username(str):
-            from_username(str):
-            json_data(str):
-            created(datetime):
-            priority_level(int):
-            trx_id(str):
-
-        Returns:
-            bool:             True on success, False otherwise
-        """
-        table = notifications_table
-        return self._create_notification(table=table, **notification)
-
-    def _wwwpoll_mark(self, nid, name, value):
-        logger.debug('wwwpoll_mark_%s: marking %s as %s', name, nid, value)
+    def wwwpoll_mark_shown(self, nid):
```

Comment:
Why repeat the same "mark as" logic 4 times with `wwwpoll_mark_shown`,`wwwpoll_mark_unshown`,`wwwpoll_mark_read`,`wwwpoll_mark_unread`?